resources/META-INF/plugin.xml

```diff
@@ -33,7 +35,15 @@
     <incompatible-with>ccom.intellij.idea.plugin.commerce.integration</incompatible-with>
 
     <extensions defaultExtensionNs="org.jetbrains.kotlin">
-        <supportsKotlinPluginMode supportsK2="true" />
+        <supportsKotlinPluginMode supportsK2="true"/>
+    </extensions>
+
+    <extensions defaultExtensionNs="com.intellij">
+        <fileType
+                name="ConfProperties"
+                extensions="conf"
```

Comment:
btw, it is interesting that https://plugins.jetbrains.com/plugin/10481-hocon uses that file format.
I'm not sure that we should rush with one more file format support, it may affect some other tools.